[examples] Add Vision Transformer Mixed PTQ example (#204)
alibaba/TinyNeuralNetwork: examples/quantization/specific/vit/readme.md

# Quantization Accuracy Results

We use post quantization to get a real INT8 VIT, the accuracy@1 result is as below:

| Model         | Top1 Acc (%) | Mixed Quantization Configuration                                         |
|---------------|--------------|--------------------------------------------------------------------------|
| Origin(FP32)  | 81.43        | -                                                                        |
| INT8          | 0            | all                                                                      |
| INT8(Mixed_1) | 78.49(-2.94) | all, except residual additions                                           |
| INT8(Mixed_2) | 80.94(-0.49) | all, except residual additions , <br/>two most quantization-sensitive fc |

1. We used [per-layer quantization analysis tool](../../layerwise_ptq_analysis.py) and observed that residual additions have a notable impact on quantization accuracy,
leading to a significant drop in accuracy. To address this, we maintained floating-point calculation for all residual additions.
While this helped to restore some accuracy, we found that there was still a considerable drop in accuracy.(-2.94).

> 2. When using [quantization error analysis tools](../../post_error_anaylsis.py), We noticed that the output activation of the MLP contains numerous outliers, which can considerably reduce the accuracy of quantization.

Reviewer: ```suggestion
2. With the help of the [quantization error analysis tools](../../post_error_anaylsis.py), We noticed that the output activation of the MLP contains numerous outliers, which can considerably reduce the accuracy of quantization.
```
